__tests__/app.test.js

```diff
@@ -262,3 +270,68 @@ describe("POST /api/articles/:article_id/comments", () => {
       })
   })
 })
+describe("PATCH /api/articles/:article_id", () => {
+  test("201: Responds with updated article vote count", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({ inc_votes: 1 })
+      .expect(201)
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(1)
+      })
+  })
+  test("201: Returns with updated article vote count", () => {
```

Comment:
This test is just a duplicate of the behaviour in the first one, so isn't required,